### PR TITLE
Fixed formatting in a table

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/welcome/developers.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Check out the following developer guides if you are looking for step-by-step ins
 
 <tr>
 <td align="left"><a href="/device-sdk/introduction/">Device SDK guide</a></td>
-<td align="left">How to integrate IoT data sources with the {{< product-c8y-iot >}} platform. The *Device SDK guide* provides detailed information on device integration using MQTT, REST and C++.</td>
+<td align="left">How to integrate IoT data sources with the {{< product-c8y-iot >}} platform. The <em>Device SDK guide</em> provides detailed information on device integration using MQTT, REST and C++.</td>
```
Comment (rssag) [inline content/welcome/developers.md:45]: in https://github.com/SoftwareAG/c8y-docs/pull/176/files it was <i>, why is it <em> here now?

Comment (JoshuaAdemola) [reply content/welcome/developers.md:45]: I always used <em>. I can change it to <i>

Comment (JoshuaAdemola) [reply content/welcome/developers.md:45]: I always used '<em>'.

Comment (JoshuaAdemola) [reply content/welcome/developers.md:45]: I always used `<em>`. I can change it to `<i>`.

Comment (rssag) [reply content/welcome/developers.md]: if we have more occurrences of `<em>`, we should use it everywhere.

**Review by BeateRixen — APPROVED** [review]

File: content/welcome/developers.md (hunk 1 above)
Comment (BeateRixen) [reply content/welcome/developers.md]: I think some developers use <em> and others use <i>. I´m afraid, we have a mix throughout the docs.

**Review by MWindel — APPROVED** [review]

**Pull request merged**